codeserver/ubi9-python-3.9/run-code-server.sh

```diff
@@ -16,6 +16,32 @@ fi
 # Initilize access logs for culling
 echo '[{"id":"code-server","name":"code-server","last_activity":"'$(date -Iseconds)'","execution_state":"running","connections":1}]' > /var/log/nginx/codeserver.access.log
 
+# Directory for settings file
+user_dir="/opt/app-root/src/.local/share/code-server/User/"
+settings_filepath="${user_dir}settings.json"
+
+json_settings='{
+  "python.defaultInterpreterPath": "/opt/app-root/bin/python3"
+}'
+
+# Check if User directory exists
+if [ ! -d "$user_dir" ]; then
+  echo "Debug: User directory not found, creating..."
```

Comment:
```suggestion
  echo "Debug: User settings directory not found, creating '$user_dir'"
```

Reply:
There is not settings directory, only the settings.json file where should be located into `/User` dir. I thing this way we mislead the debugging

Reply:
Oh, right - I overlooked, sorry. Then no `settings` addition here, agree.

Reply:
This applies also to the rest of the comments, shall i resolve them?

Reply:
I removed the one that applied, otherwise it's about having actual value directly in the debug message, which is a different thing, I suppose :slightly_smiling_face:

Reply:
fair!